task06/routes/cart.route.ts

```diff
@@ -0,0 +1,29 @@
+import express from 'express';
+import { Request, Response } from 'express';
+import {
+  getCartByUserId,
+  removeCart,
+  updateCart,
+} from '../controller/cart.controller';
+import { createOrder } from '../controller/order.controller';
+import { authenticateUser } from '../controller/user.controller';
+
+const router = express.Router();
+router.use(authenticateUser);
+
+router
+  .route('/')
+  .get((req: Request, res: Response) => {
+    getCartByUserId(req, res);
+  })
```

Comment:
I assume you can simplify the code below.

```suggestion
  .get(getCartByUserId)
```

Reply:
Fixed